docs/static/ls-ls-http.asciidoc

```diff
@@ -8,3 +8,126 @@ NOTE: Check out these <<http-considerations,considerations>> before you implemen
 
 For now, <<plugins-outputs-http,http output>> to <<plugins-inputs-http,http input>> with manual configuration may be the best path forward if these limitations don't apply to your use case.
 
+==== Configuration overview
```

Comment:
```suggestion
[[overview-http-http]]
==== Configuration overview
```

Reply:
I know that there's not one for the lumberjack session, but we need to add anchors for all new content.  Shorter is better because people aren't likely going to try to guess the URL.